### PR TITLE
Update Dist_rmr557.py

### PR DESCRIPTION
**Pull request by RashedRifat**
Created new pdf 1/14(3x^2 - 6x + 9) and filled in the appropriate data

### CODE REVIEW

File: distributions/Dist_rmr557.py [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
 	def mean(self):
 		"""This is the mean of the PDF"""
-		return 0.
+		return 1
 
 	def std(self):
 		"""This is the standard deviation of the pdf"""
-		return np.sqrt(0.5)
\ No newline at end of file
+		return np.sqrt(455) / 35
```
Comment (cranmer) [inline distributions/Dist_rmr557.py:21]: I got sqrt(13/35)
https://www.wolframalpha.com/input/?i=Integrate%5B%28x-1%29%5E2*%281%2F14%29*%283*%28x**2%29+-+6*x+%2B+9%29%2C+%7Bx%2C0%2C2%7D%5D

File: distributions/Dist_rmr557.py [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
-		self.x_max = 1
+		self.f_max = 9.0/14
+		self.x_min = 0
+		self.x_max = 2
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return np.abs(x)
+		return (1.0/14)(3.0*(x**2) - 6.0*x + 9.0)
```
Comment (cranmer) [inline distributions/Dist_rmr557.py]: you have a python syntax problem here. you need (1.0/14)*(3.0*...

**Comment by RashedRifat** [comment]
Note that for the standard deviation, sqrt(455)/35 = sqrt(12/35). WolframAlpha may state that the standard deviation is sqrt(12/35); I simply rewrote that in an alternate version.

**Comment by cranmer** [comment]
using new PR to avoid conflicts

**Pull request closed without merging**